benchmark/intl_stream_benchmark.dart

```diff
@@ -0,0 +1,77 @@
+import 'dart:math';
+
+// Import BenchmarkBase class.
+import 'package:benchmark_harness/benchmark_harness.dart';
+
+const int asciiZeroCodeUnit = 2;
+const int listlength = 10000;
+
+// Create a new benchmark by extending BenchmarkBase
+class NewMethod extends BenchmarkBase {
+  late String string;
+  late String string2;
+  int zeroDigit = 15;
+  NewMethod() : super('New');
+
+  static void main() {
```

Comment:
This is not a real issue, but `main` as a method name has special significance in Dart; perhaps rename to something like `runBenchmark()`?

Reply:
Delete `main`.
Most of our benchmark's call `report()` method from outside.